client/src/pages/index.tsx

```diff
@@ -23,6 +25,8 @@ export default function Home() {
      */
   }
 
+  const eventId = 1;
+
```

Comment:
Don't have much context but I'm assuming this will be changed so that it is dynamic?

Reply:
Yes, the Rsvp modal will be under the event page, where the id will be taken from the URL instead.

Reply:
Yep, I'll make sure I pass it as a prop.